### DISCUSSION TITLE
vue-cli-plugin-tauri for Vite

### DISCUSSION THREAD

**Discussion by tcastelly** [tauri-apps/tauri#1919] [Ideas]
The vue-cli-plugin-tauri works perfectly. But I'm wondering if a [Vite](https://github.com/vitejs/vite) plugin is something conceivable?
It can be useful for React projects or tsx support.

**Comment by amrbashir** [comment]
you can use create-tauri-app to make a vite project with tauri

**Reply by tcastelly** [reply]
Amazing! I'm sorry I did not noticed it was possible with `create-tauri-app`.

I had some troubles during the creation of a project. I created an issue:
https://github.com/tauri-apps/tauri/issues/1924

But in the end it works :) thank you.
